components/Table/Table.js

```diff
@@ -3,61 +3,95 @@ export class Table {
   tableContainer = null;
   tableBody = null;
   tableHeader = null;
+  callbackFunc = null;
+  isOrderAsc = false;
 
-  constructor(columnsConfig, tableContainer) {
+  constructor(columnsConfig, tableContainer, callbackFunc) {
     this.columnsConfig = columnsConfig;
     this.tableContainer = tableContainer;
+    this.callbackFunc = callbackFunc;
   }
 
   createTable() {
-    this.tableHeader = document.createElement("div");
-    this.tableHeader.classList.add("table-header");
+    this.tableHeader = document.createElement('div');
+    this.tableHeader.classList.add('table-header');
     this.tableContainer.append(this.tableHeader);
 
-    this.tableBody = document.createElement("div");
-    this.tableBody.classList.add("table-body");
+    this.tableBody = document.createElement('div');
+    this.tableBody.classList.add('table-body');
     this.tableContainer.append(this.tableBody);
 
     this.createHeader();
   }
 
   createHeader() {
-    const tableHeaderRow = document.createElement("div");
-    tableHeaderRow.classList.add("table-header-row");
+    const tableHeaderRow = document.createElement('div');
+    tableHeaderRow.classList.add('table-header-row');
 
     const cells = this.columnsConfig.map((col) => {
-      const cell = document.createElement("div");
-      cell.classList.add("table-header__cell");
+      const cell = document.createElement('div');
+      cell.classList.add('table-header__cell');
+      cell.setAttribute('data-key', col.key);
+      cell.setAttribute('data-sortable', col.sortable);
+      const headerLabel = document.createElement('div');
+      headerLabel.classList.add('table-header-label');
 
-      const actionsContainer = document.createElement("div");
-      actionsContainer.classList.add("actions-container");
+      if (col.sortable) {
+        headerLabel.addEventListener('click', () => {
+          this.isOrderAsc = !this.isOrderAsc;
+          const dataKey = cell.getAttribute('data-key');
+          arrowBtn.setAttribute('asc', this.isOrderAsc);
+          this.callbackFunc(dataKey, this.isOrderAsc);
+        });
+      }
 
-      const arrowBtn = document.createElement("span");
-      arrowBtn.classList.add("material-icons");
-      arrowBtn.textContent = "south";
+      const actionsContainer = document.createElement('div');
+      actionsContainer.classList.add('actions-container');
 
-      const menuBtn = document.createElement("span");
-      menuBtn.classList.add("material-icons");
-      menuBtn.textContent = "more_vert";
+      const arrowBtn = document.createElement('span');
+      arrowBtn.classList.add('material-icons');
+      arrowBtn.textContent = 'south';
+      arrowBtn.setAttribute('name', 'arrow');
```

Comment:
Should it really have the `name` attribute?

Reply:
Let's avoid it to not make overhead css selectors, instead of it we could add class for the arrow button